canopen/node/remote.py

```diff
@@ -142,7 +137,8 @@ def __load_configuration_helper(self, index, subindex, name, value):
             if e.code != 0x06010002:
                 # Abort codes other than "Attempt to write a read-only object"
                 # should still be reported.
-                logger.warning('[ERROR SETTING object {0:#06x}:{1:#06x}]  {2}'.format(index, subindex, str(e)))
+                logger.warning('[ERROR SETTING object %#06x:%#06x]  %s',
+                               index, subindex, str(e))
```

Comment:
The `%s` format specifier will implicitly convert the argument to a string.

```suggestion
                               index, subindex, e)
```